optimisations/README.md

```diff
@@ -0,0 +1,6 @@
+# Optimisations
+
+Please select your operating system below to see optimisations available for your system.
+
+- [macOS](mac)
```

Comment:
Should we delete this for now? Just in case it causes confusion?

Reply:
yeah sure